feature/gribi/ate_tests/base_hierarchical_route_installation_test/base_hierarchical_route_installation_test.go

```diff
@@ -408,39 +408,14 @@ func TestRecursiveIPv4Entries(t *testing.T) {
 		fn   func(t *testing.T, args *testArgs)
 	}{
 		{
-			name: "TestInvalidIPv4Entry",
```

Comment:
Seems like you cleaned up and improved the code logic. Please help update the PR description to help reviewers.